README.md

```diff
@@ -1,30 +1,17 @@
 # LPS follow-up plan API: Consumer guide
 
-This API allows for the submission of a follow-up plan to NAV and/or the general practitioner on behalf of an employer.
+This API allows for the submission of a [follow-up plan (oppfølgingsplan)](https://www.nav.no/arbeidsgiver/oppfolgingsplan) to NAV and/or the general practitioner on behalf of an employer.
 This is a <i>Delegated
 API</i>, which means that you as the API consumer are acting on behalf of another company/end-user. The API is secured
-with Maskinporten.
+with Maskinporten. NAV does not authorize the API consumer directly. Authorization to submit a plan is given to you as an API consumer by the end-user in Altinn.
 
-When a follow-up plan is submitted, the API will generate a PDF file to be shared with NAV and/or the general
-practitioner, depending on the input from `sendPlanToNav` and `sendPlanToGeneralPractitioner`. When `sendPlanToNav` is
-set
-to true, the resulting PDF is accessible to the employee on sick leave at
-https://person.nav.no/dokumentarkiv/tema/OPP (login required). If it is set to false, the follow-up plan will not be
-shared with NAV, and the PDF will not be added to the NAV website. When `sendPlanToGeneralPractitioner` is set to true,
-the resulting PDF will be sent to the general practitioner.
```

Comment:
Det greiene her kunne vi kanskje puttet inn i skjemaet i Swagger i stedet for, på de relevante feltene.

Reply:
Sant det :)